general/releases.rst

```diff
@@ -0,0 +1,178 @@
+
+.. _releases_changelog:
+
+Changelog
+*********
+The changelog is stored in the :ref:`optee_os` git (CHANGELOG.md_). There you
+can see what has been done between the different releases in terms of commits as
+well as pull requests.
+
+.. _releases_versioning_schema:
+
+Versioning schema
+*****************
+OP-TEE follows `Semantic Versioning 2.0.0`_. What that means in practice is well
+described at the link just shown.
+
+.. _releases_release_procedure:
+
+Release procedure
+*****************
+There are certain steps that needs to be done when making a release. This
+checklist here serves as guidance to the one in charge of making a new release.
+Roughly start with this 2-3 weeks before the targeted release date.
+
+tl;dr
+=====
+.. list-table:: Short version of the OP-TEE release procedure
+    :widths: 60 300 10
+    :header-rows: 1
+
+    * - When
+        (Tminus)
+      - Action
+      - Example
+
+    * - 3w
+      - Create release pull request
+      - `PR#3099`_
+
+    * - 3w
+      - Inform maintainers about upcoming release
+      -
+
+    * - 1w
+      - Create release candidate tag in optee_* + build.git
+      - git tag -a 3.x.y-rc1 -m "3.x.y-rc1"
+
+    * - 1w
+      - Let maintainers know about the release candidate tag
+      -
+
+    * - 1w
+      - Test platform builds / devices
+      -
+
+    * - Release day
+      - Update CHANGELOG.md_
+      - `changelog example`_
+
+    * - Release day
+      - Collect/merge ``Tested-By`` tags
+      - `commit example`_
+
+    * - Release day
+      - Create release tag in optee_* + build.git
+      - git tag -a 3.x.y -m "3.x.y"
+
+    * - Release day
+      - Create release branch in :ref:`manifest`
+      - git checkout -b 3.x.y origin/master
+
+    * - Release day
+      - Update manifest XML-files
+      - `3.6.0 stable`_
+
+    * - Release day
+      - Inform maintainers and stakeholder that release has been completed.
+      -
+
+
+Long version
+============
+
+    1. Create a "release pull request" at GitHub ought to collect ``Tested-By``
+       tags from various maintainers. As an example, see `PR#3099`_.
+
+    2. Send email to all maintainers to let them know about the upcoming
+       release. The addresses to the maintainers can be found in the
+       MAINTAINERS_ file.
+
+       .. hint::
+            With this command in bash you will get all email addresses
+
+            .. code-block:: bash
+
+                $ cat MAINTAINERS | grep '[RM]:.*<.*@.*>' | \
+                  sed 's/>.*/>/' | sed 's/.:\t//' | sort | uniq
+
+    3. Create a release candidate (RC) tag (annotated tag, i.e., ``git tag -a
+       3.x.y-rc1 -m "3.x.y-rc1"``) in the following gits
+       ``optee_*`` and ``build.git``. One way to do it is like this
+
+       .. code-block:: bash
+
+            $ export VER=3.x.y-rc1
+            $ for d in optee* build; do ( cd $d; git tag -a $VER -m $VER ); done
+            $ for d in optee* built; do ( cd $d; git push origin $VER ); done
```

Comment:
s/built/build/

Reply:
Will fix when force push and adding your tag! Thanks for feedback + review!